File: 25000~25999/25600~25699/25640.cpp
/*
  [25640: MBTI](https://www.acmicpc.net/problem/25640)

  Tier: Bronze 4
  Category: 문자열, 구현
*/
#include <bits/stdc++.h>

using namespace std;
string s, s2;
int N, ans;

int main() {
  ios::sync_with_stdio(0);cin.tie(0);cout.tie(0);

  cin >> s >> N;

  while(N--) {
    cin >> s2;

    if(s == s2) ans++;
  }

  cout << ans;
}

File: 25000~25999/25600~25699/25640.py
"""
[25640: MBTI](https://www.acmicpc.net/problem/25640)

Tier: Bronze 4
Category: 구현
"""


def solution():
  ans = 0
  s = input()

  for _ in range(int(input())):
    if s == input():
      ans += 1

  return ans


if __name__ == '__main__':
  print(solution())
